addons/l10n_de/models/ir_attachment.py

```diff
@@ -17,6 +17,7 @@ def _except_audit_trail(self):
             attachment.res_model == 'account.move'
             and attachment.res_id
             and attachment.raw
+            and attachment.company_id.check_account_audit_trail                                        
```

Comment:
ci/style fails because of the white spaces you leave at the end of your lines, could you remove them please ?

```suggestion
            and attachment.company_id.check_account_audit_trail
```